packing.gradle

```diff
@@ -21,13 +21,14 @@ dependencies {
     // Capsule manages the fat jar building process
     capsule 'io.nextflow:capsule:1.1.1'
     capsule 'io.nextflow:capsule-maven:1.0.3.2'
-    console project(':plugins:nf-console')
-    ga4gh   project(':plugins:nf-ga4gh')
-    google  project(':plugins:nf-google')
-    amazon  project(':plugins:nf-amazon')
-    azure   project(':plugins:nf-azure')
-    tower   project(':plugins:nf-tower')
-    wave    project(':plugins:nf-wave')
+    amazon    project(':plugins:nf-amazon')
+    azure     project(':plugins:nf-azure')
+    console   project(':plugins:nf-console')
+    ga4gh     project(':plugins:nf-ga4gh')
+    google    project(':plugins:nf-google')
+    pathcache project(':plugins:nf-path-cache')
```

Comment:
No really needed adding it here, for example nf-codecommit plugin is not include in this list